proto/stream_plan.proto

```diff
@@ -151,6 +151,11 @@ message AggCallState {
   // use the one column of stream's result table as the AggCall's state, used for count/sum/append-only extreme.
   message AggResultState {}
 
+  // use untransformed result as the AggCall's state, used for append-only approx count distinct.
+  message AggRegisterState {
```

Comment:
I think it'll be better to make this `AggCallState` variant more general. Any agg functions that reply on a single state table whose schema is deduced by frontend and backend with implicit consensus can use this variant.

A suggestion would be just `TableState`.

Reply:
> a single state table whose schema is deduced by frontend and backend with implicit consensus

fixed